homeassistant/components/prowl/__init__.py

```diff
@@ -1 +1,47 @@
 """The prowl component."""
+
+import logging
+
+import prowlpy
+
+from homeassistant.const import CONF_API_KEY
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryError, ConfigEntryNotReady
+
+from .const import DOMAIN, PLATFORMS
+from .helpers import async_verify_key
+from .notify import PLATFORM_SCHEMA, ProwlConfigEntry
+
+_LOGGER = logging.getLogger(__name__)
+
+__all__ = ["DOMAIN", "PLATFORM_SCHEMA"]
+
+
+# This is just to suppress https://github.com/home-assistant/core/blob/dev/homeassistant/setup.py#L365-L378
+async def async_setup(hass: HomeAssistant, entry: ProwlConfigEntry) -> bool:
+    """Set up the Prowl component."""
+    hass.data.setdefault(DOMAIN, {})
+    return True
```

Comment:
Instead of this, I think we should do like this:
```suggestion
CONFIG_SCHEMA = cv.platform_only_config_schema(DOMAIN)
```